packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFInstanceManager_Test.h

```diff
@@ -7,6 +7,11 @@
 NS_ASSUME_NONNULL_BEGIN
 
 @interface FWFInstanceManager ()
+/**
+ * The next identifier that will be used for a host-created instance.
+ */
+@property long nextIdentifier;
```

Comment:
I'm not a fan of this pattern, but it is fine for now. https://github.com/flutter/flutter/issues/134777 should remove the need for it.